src/point.rs

```diff
@@ -1,9 +1,8 @@
 use crate::streamed_data::StreamedData;
 use crate::streamed_data::StreamedState;
-use crate::streamed_data::*;
 
 #[derive(Ord, PartialOrd, Eq, PartialEq, Debug, Copy, Clone)]
-pub struct Timeval(pub i32, pub i32);
+pub struct Timeval(pub i64, pub i64);

```

Comment:
i'm not sure why, but these types weren't working on my system

Reply:
It's bindings for libc so probably platform specific, yes. Not sure how to deal with this. Maybe convert to a Rust time/duration type to avoid this kind of trouble. Maybe use usize to be platform compatible.

Reply:
sounds good, made it an isize